services/comparison.py

```diff
@@ -953,7 +940,10 @@ def change_coverage(self) -> Optional[float]:
             and self.head_coverage
             and self.head_coverage.coverage
         ):
-            return float(self.head_coverage.coverage - self.base_coverage.coverage)
+            return float(
```

Comment:
This could result in some weird behavior if head_coverage.coverage was a non-float/int value

Reply:
Since you can't subtract 2 strings

<img width="1162" alt="Screenshot 2024-07-31 at 4 34 48 PM" src="https://github.com/user-attachments/assets/72645d0d-dc73-4bce-9b11-c9e79bd078f6">